.gitignore

```diff
@@ -21,9 +21,13 @@
 # temporary folders
 1_wqp_pull/tmp/*
 
+
 # old folder that stores junk
 old/*
 
 # ignore geodb for national network
 # and delaware shape file
 8_delaware/in/*
+
+#ignore out files
+out/
```

Comment:
I'm not seeing you write any files to out/? Files in the out folders should be handled by ignoring the .rds and other file types.

Reply:
I created some plots and maps which I replaced in the out file.